src/Evolution/Systems/Ccz4/BoundaryConditions/BoundaryCondition.hpp

```diff
@@ -7,9 +7,7 @@
 
 #include "Domain/BoundaryConditions/BoundaryCondition.hpp"
 
-namespace Ccz4 {
-/// \brief Boundary conditions for the Ccz4 system
```

Comment:
You should keep the documentation, though. You can write dox on the new line 10:
```cpp
/// \brief Boundary conditions for the Ccz4 system
namespace Ccz4::BoundaryConditions {
```
otherwise things don't always show up in doxygen and that can be annoying